src/hera.cpp

```diff
@@ -398,6 +398,14 @@ evmc_set_option_result hera_set_option(
     return EVMC_SET_OPTION_SUCCESS;
   }
 
+  if (strcmp(name, "benchmark") == 0) {
```

Comment:
So we would run testeth with command line argument `benchmarking=true`?

Reply:
Yes, by `--evmc benchmark=true`.